src/main/java/org/folio/consortia/service/impl/UserTenantServiceImpl.java

```diff
@@ -79,4 +127,75 @@ public UserTenantCollection getByUsernameAndTenantId(UUID consortiumId, String u
     result.setTotalRecords(1);
     return result;
   }
+
+  private User prepareShadowUser(UUID userId, UserTenant userTenantDto) {
+    User userOptional = getUser(userId);
+    User user = new User();
+    if (Objects.nonNull(userOptional.getId())) {
+      user.setId(UUID.randomUUID().toString());
+      user.setPatronGroup(PATRON_GROUP);
+      user.setUsername(userOptional.getUsername() + HelperUtils.randomString(RANDOM_STRING_COUNT));
+      if(Objects.nonNull(userOptional.getPersonal())) {
+        Personal personal = new Personal();
+        personal.setLastName(userOptional.getPersonal().getLastName());
+        personal.setFirstName(userOptional.getPersonal().getFirstName());
+        personal.setEmail(userOptional.getPersonal().getEmail());
+        personal.setPreferredContactTypeId(userOptional.getPersonal().getPreferredContactTypeId());
+        user.setPersonal(personal);
+      }
+      user.setPatronGroup(userOptional.getPatronGroup());
+      user.setActive(IS_PRIMARY_TRUE);
```

Comment:
isActive and IS_PRIMARY_TRUE have different meaning, so we can write just true here

Reply:
fixed